src/model/assignments.model.ts

```diff
@@ -0,0 +1,49 @@
+import { Entity, Column, PrimaryGeneratedColumn, CreateDateColumn, UpdateDateColumn, DeleteDateColumn } from 'typeorm'
+
+@Entity('assignments')
+export default class Assignment {
+  @PrimaryGeneratedColumn()
+  id: number
+
+  @Column()
+  course_id: number
+
+  @Column({ length: 128 })
+  name: string
+
+  @Column()
+  start_date: Date
```

Comment:
I do agree that the column names should be in `snake_case` (as per postgres syntax), but we're going to want them in camel case in the API (as per TS/JS convention). 

To do this add the `name` param to the `@Column` decorators.

Eg:

```
@Column({ name: 'start_date' }
startDate: Date
```

(Sorry we didn't do this in the initial commit, but it'll be fixed with the Auth PR for users).

Reply:
Make sure to do this to all columns you've added, not just this one.